app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -81,17 +81,18 @@ private void nextActivity() {
             final KeyPair keyPair = profile.getKeyPairs().get(0);
 
 
-            LOGGER.info("Sending my profile with name: " + profile.getName() + ", public key: "
-                    + keyPair.getPublicKey().toString());
+            LOGGER.info("Sending my profile with name: " + profile.getName() + ", public key: " + keyPair.getPublicKey()
```

Comment:
If you transform logging calls to the following, they're a lot more efficiënt because you don't have to concatenate all those Strings together:
`LOGGER.info("Sending my profile with name: {}, public key: {}, iban: {}", profile.getName(), keyPair.getPublicKey(), profile.getIban());`